lib/cisco_node_utils/radius_server.rb

```diff
@@ -39,6 +39,24 @@ def initialize(name, instantiate=true)
       end
       @name = name
 
+      if platform == :ios_xr
```

Comment:
Yes. For Nexus, if the ports are specified in the constructor, the setter methods are called for each port to use the constructor-provided values.